### ISSUE TITLE
Not working in Visual Studio Professional 2022 and ReSharper 2023.1

### ISSUE THREAD

**Issue by jerone** [matkoch/resharper-verify#21]
I've installed this extension via ReSharper in Visual Studio and I've never seen it working.
There is never an context menu actions available:

![image](https://user-images.githubusercontent.com/55841/231458237-8e3fdab3-b713-4a68-bfd1-82f6be8a5ef3.png)

Verify by itself works good.

Information:
* Microsoft Windows 11 Enterprise
* Microsoft Visual Studio Professional 2022 (64-bit) - Version 17.5.4 (and a few versions earlier)
* ReSharper 2023.1 (and a few versions earlier)
* Verify Support (this extension) 2022.3.0
* Verify.MSTest 19.8.3
* MSTest 3.0.2

**Comment by SimonCropp** [comment]
can u try v 11.3.0 of diffengine

**Comment by matkoch** [comment]
@SimonCropp not entirely sure where this is coming from, but with MSTest there's an additional line that is not recognized by Verify.ExceptionParsing.

<img width="1234" alt="Parallels Desktop-Windows 11-2023-04-14-y7dLgV0v@2x" src="https://user-images.githubusercontent.com/5005566/232109055-d00b3480-e793-4605-8405-01b893c5872a.png">

Sample message:

```
Test method TestProject1.UnitTest1.TestMethod1 threw exception: 
VerifyException: Directory: C:\Users\matt\source\repos\TestProject1\TestProject1
NotEqual:
  - Received: UnitTest1.TestMethod1.received.txt
    Verified: UnitTest1.TestMethod1.verified.txt

FileContent:

NotEqual:

Received: UnitTest1.TestMethod1.received.txt
foo
Verified: UnitTest1.TestMethod1.verified.txt
```

**Comment by SimonCropp** [comment]
this should fix it https://github.com/matkoch/resharper-verify/pull/22

**Comment by jerone** [comment]
Just wondering what the status is of this issue. 
Can reproduce the same issue with the latest updates to date.

**Comment by SimonCropp** [comment]
can u try 20.1.1

**Comment by matkoch** [comment]
There was no new release since I last tried it

**Comment by SimonCropp** [comment]
@jerone so @matkoch needs to update this project to consume that version of the verify exception parsing

**Comment by matkoch** [comment]
Should be fixed with 2023.1.1

**Comment by jerone** [comment]
Thank you, got it working ✅
